src/components/topbar/Topbar.jsx

```diff
@@ -47,6 +50,26 @@ const Topbar = (props) => {
       </Row>
     </div>
   );
+
+  const handleLogout = () =>{
+    firebase.auth().signOut().catch(error => console.log(error))
```

Comment:
Don't need the firebase signOut. Just clear the localstorage and redirect to signin page